templates/intro-services/README.ipynb

```diff
@@ -9,23 +9,23 @@
     "**⏱️ Time to complete**: 5 min (15 on GCP)\n",
     "\n",
     "This tutorial shows you how to:\n",
-    "1. Develop a simple Ray Serve application locally.\n",
-    "2. Deploy the application to production as an Anyscale service.\n",
-    "3. Monitor the production application.\n",
+    "1. Develop a simple Ray Serve app in a workspace.\n",
+    "2. Deploy the app to production as an Anyscale Service.\n",
+    "3. Monitor the production app.\n",
     "4. Configure service scaling.\n",
     "\n",
-    "**Note**: This tutorial is run within a workspace. Please overview the `Introduction to Workspaces` template first before this tutorial."
+    "**Note**: This tutorial runs in a workspace. See `Intro to Workspaces` before running this tutorial."
```

Comment:
```suggestion
    "**Note**: This tutorial runs in a workspace. See the `Intro to Workspaces` template before running this tutorial."
```
or point at a docs page (with a link), otherwise i'm not sure where to go here